jaeger/static/generate.go

```diff
@@ -0,0 +1,21 @@
+// +build ignore
+
+package main
+
+import (
+	"github.com/linkerd/linkerd2/jaeger/static"
+	"github.com/shurcooL/vfsgen"
+	log "github.com/sirupsen/logrus"
+)
+
+func main() {
+	err := vfsgen.Generate(static.Templates, vfsgen.Options{
+		Filename:     "generated_templates.gogen.go",
```

Comment:
Nit: Can you name this something different, like `generated_jaeger_templates.gogen.go` so that the output from `bin/build-cli-bin` is a little more expressive? Currently we have:
```bash
$ bin/build-cli-bin 
writing generated_templates.gogen.go
writing generated_templates.gogen.go
target/cli/linux-amd64/linkerd-jaeger
```
(would be best if the other generated file for the linkerd binary is renamed as well...)